agent/engine/task_manager.go

```diff
@@ -323,6 +324,19 @@ func (mtask *managedTask) handleContainerChange(containerChange dockerContainerC
 	}
 }
 
+// releaseIPInIPAM releases the ip used by the task for awsvpc
+func (mtask *managedTask) releaseIPInIPAM() {
+	if mtask.ENI == nil {
+		return
+	}
+	seelog.Infof("Releasing ip in IPAM for task used eni, task: %s", mtask.Task.Arn)
+
+	err := mtask.engine.releaseIPInIPAM(mtask.Task)
+	if err != nil {
+		seelog.Infof("Releasing the ip in IPAM failed, task: [%s], err: %v", mtask.Task.Arn, err)
```

Comment:
Why is this `Infof`? And, why is it ok to consume this error?

Reply:
For this kind of error, it's fine because there will be  more available ips than the task, so if one ip was failed to release the next task can still get an available ip. But in rare condition all the release failed, and all the ip has been used then no task can be started, that's why I marked it as `Infof`, any better suggestion here?

Reply:
So, it's probably fine that we consume this error and move on. I'd still like this to be logged as a warning.